src/Components/Cards/Cards.tsx

```diff
@@ -0,0 +1,235 @@
+import React from "react";
+import { useState, useRef, useEffect } from "react";
```

Comment:
아래처럼 한 줄로 작성할 수도 있습니다! 

```suggestion
import React, { useState, useRef, useEffect } from "react";
```